model/event/DeliveryCreatedEvent.php

```diff
@@ -55,4 +55,22 @@ public function jsonSerialize()
             'delivery' => $this->deliveryUri
         ];
     }
+
+    /**
+     * @return string
+     */
+    public function getWebhookEventName()
```

Comment:
if the WEBHOOK_EVENT_NAME is used only here, i think is safe to return the value,
if it is used inside others let's remove this function and change the visibility of the const to public
the extra code does not help to read faster the code